### PR TITLE
Update API Core details in WORKING-GROUPS.md

### PR DESCRIPTION
**Pull request by dprotaso**
Point to the correct drive folder and meeting notes

### CODE REVIEW

File: working-groups/WORKING-GROUPS.md [hunk 1]
```diff
@@ -77,8 +77,8 @@ API
 | Forum                      | [knative-dev@](https://groups.google.com/forum/#!forum/knative-dev)                                                                                             |
 | Community Meeting VC       | See the top of the [Meeting notes](https://docs.google.com/document/d/1NC4klOdNaU-N-PsKLyXBqDKgNSHtxCDep29Ta2b5FK0/edit)                                        |
```
Comment (duglin) [inline working-groups/WORKING-GROUPS.md:78]: here too

**Comment by evankanderson** [comment]
/lgtm
/approve

Sorry I missed that one

**Pull request merged**